Makefile

```diff
@@ -49,9 +49,6 @@ install-dev-deps:
 	pip3 install --user pycodestyle==2.5.0
 	go get -v golang.org/x/lint/golint github.com/mjibson/esc golang.org/x/tools/cmd/goimports
 
-docker-build:
-	docker build -t srcd/identity_matching .
-
 docker-test: docker-build
```

Comment:
Shouldn't `docker-build` be removed from here then?

Reply:
From where? :thinking:

Reply:
Sorry, I got it now. It should be left because the dependency is still relevant: we cannot test if the image is not built.